views/containers/GameLoader.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import Game from '../components/Game';
+import LoadingScreen from '../components/LoadingScreen';
+import * as BrowserActions from '../actions/Browser';
+
+class GameLoaderComponent extends React.Component {
+  constructor() {
+    super();
+    this.didLoad = false;
```

Comment:
Use the inner state of the component, doing so will trigger a re-render when it changes.

```javascript
this.state =  {
  didLoad: false,
}
```